{
  "uid" : "abbaa4803039c0e8",
  "name" : "POST delete_db",
  "fullName" : "tests_api.test_bd.TestPOST#test_delete_db",
  "historyId" : "aacf29e4b56c0d304e9b651197b003cc",
  "time" : {
    "start" : 1726535362132,
    "stop" : 1726535362132,
    "duration" : 0
  },
  "status" : "skipped",
  "statusMessage" : "XFAIL When using this method during a test run, the database may be in deleting status.\n\nrequests.exceptions.JSONDecodeError: Expecting value: line 1 column 1 (char 0)",
  "statusTrace" : "self = <Response [502]>, kwargs = {}\n\n    def json(self, **kwargs):\n        r\"\"\"Returns the json-encoded content of a response, if any.\n    \n        :param \\*\\*kwargs: Optional arguments that ``json.loads`` takes.\n        :raises requests.exceptions.JSONDecodeError: If the response body does not\n            contain valid json.\n        \"\"\"\n    \n        if not self.encoding and self.content and len(self.content) > 3:\n            # No encoding set. JSON RFC 4627 section 3 states we should expect\n            # UTF-8, -16 or -32. Detect which one to use; If the detection or\n            # decoding fails, fall back to `self.text` (using charset_normalizer to make\n            # a best guess).\n            encoding = guess_json_utf(self.content)\n            if encoding is not None:\n                try:\n                    return complexjson.loads(self.content.decode(encoding), **kwargs)\n                except UnicodeDecodeError:\n                    # Wrong UTF codec detected; usually because it's not UTF-8\n                    # but some other 8-bit codec.  This is an RFC violation,\n                    # and the server didn't bother to tell us what codec *was*\n                    # used.\n                    pass\n                except JSONDecodeError as e:\n                    raise RequestsJSONDecodeError(e.msg, e.doc, e.pos)\n    \n        try:\n>           return complexjson.loads(self.text, **kwargs)\n\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/requests/models.py:974: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/json/__init__.py:346: in loads\n    return _default_decoder.decode(s)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/json/decoder.py:337: in decode\n    obj, end = self.raw_decode(s, idx=_w(s, 0).end())\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <json.decoder.JSONDecoder object at 0x7fec3f2f2930>\ns = '<html>\\r\\n<head><title>502 Bad Gateway</title></head>\\r\\n<body>\\r\\n<center><h1>502 Bad Gateway</h1></center>\\r\\n<hr><center>nginx/1.18.0 (Ubuntu)</center>\\r\\n</body>\\r\\n</html>\\r\\n'\nidx = 0\n\n    def raw_decode(self, s, idx=0):\n        \"\"\"Decode a JSON document from ``s`` (a ``str`` beginning with\n        a JSON document) and return a 2-tuple of the Python\n        representation and the index in ``s`` where the document ended.\n    \n        This can be used to decode a JSON document from a string that may\n        have extraneous data at the end.\n    \n        \"\"\"\n        try:\n            obj, end = self.scan_once(s, idx)\n        except StopIteration as err:\n>           raise JSONDecodeError(\"Expecting value\", s, err.value) from None\nE           json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)\n\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/json/decoder.py:355: JSONDecodeError\n\nDuring handling of the above exception, another exception occurred:\n\n    @pytest.fixture()\n    def get_token():\n        \"\"\"\n        Method return:\\n\n        token: str\\n\n        body: dict\\n\n        new_password: str\\n\n        old_password: str\\n\n        email: str\n        :returns: token: str, body: dict, new_password: str, old_password:str, email: str\n        \"\"\"\n        load_dotenv()\n        email = os.getenv('EMAIL')\n        old_password = os.getenv('PASSWORD')\n        new_password = '123456789'\n        try:\n            body = {\"email\": email, \"password\": f'{old_password}'}\n            result = requests.post('https://dbend.areso.pro/login', json=body)\n            token = result.json()['token']\n            if token != {}:\n                new_password, old_password = old_password, new_password\n            Checking.check_status_code(result, 200)\n            return token, body, new_password, old_password, email\n        except Exception as ex:\n            print(ex)\n            old_password, new_password = new_password, old_password\n            body = {\"email\": email, \"password\": f'{old_password}'}\n            result = requests.post('https://dbend.areso.pro/login', json=body)\n>           token = result.json()['token']\n\nconftest.py:40: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <Response [502]>, kwargs = {}\n\n    def json(self, **kwargs):\n        r\"\"\"Returns the json-encoded content of a response, if any.\n    \n        :param \\*\\*kwargs: Optional arguments that ``json.loads`` takes.\n        :raises requests.exceptions.JSONDecodeError: If the response body does not\n            contain valid json.\n        \"\"\"\n    \n        if not self.encoding and self.content and len(self.content) > 3:\n            # No encoding set. JSON RFC 4627 section 3 states we should expect\n            # UTF-8, -16 or -32. Detect which one to use; If the detection or\n            # decoding fails, fall back to `self.text` (using charset_normalizer to make\n            # a best guess).\n            encoding = guess_json_utf(self.content)\n            if encoding is not None:\n                try:\n                    return complexjson.loads(self.content.decode(encoding), **kwargs)\n                except UnicodeDecodeError:\n                    # Wrong UTF codec detected; usually because it's not UTF-8\n                    # but some other 8-bit codec.  This is an RFC violation,\n                    # and the server didn't bother to tell us what codec *was*\n                    # used.\n                    pass\n                except JSONDecodeError as e:\n                    raise RequestsJSONDecodeError(e.msg, e.doc, e.pos)\n    \n        try:\n            return complexjson.loads(self.text, **kwargs)\n        except JSONDecodeError as e:\n            # Catch JSON-related errors and raise as requests.JSONDecodeError\n            # This aliases json.JSONDecodeError and simplejson.JSONDecodeError\n>           raise RequestsJSONDecodeError(e.msg, e.doc, e.pos)\nE           requests.exceptions.JSONDecodeError: Expecting value: line 1 column 1 (char 0)\n\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/requests/models.py:978: JSONDecodeError",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "get_token",
    "time" : {
      "start" : 1726535362132,
      "stop" : 1726535363076,
      "duration" : 944
    },
    "status" : "broken",
    "statusMessage" : "requests.exceptions.JSONDecodeError: Expecting value: line 1 column 1 (char 0)\n",
    "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/Oblache/Oblache/conftest.py\", line 40, in get_token\n    token = result.json()['token']\n            ^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/requests/models.py\", line 978, in json\n    raise RequestsJSONDecodeError(e.msg, e.doc, e.pos)\n",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : true,
    "hasContent" : true
  } ],
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "suite",
    "value" : "POST"
  }, {
    "name" : "tag",
    "value" : "@pytest.mark.xfail(reason='When using this method during a test run, the database may be in deleting status.')"
  }, {
    "name" : "parentSuite",
    "value" : "tests_api"
  }, {
    "name" : "subSuite",
    "value" : "TestPOST"
  }, {
    "name" : "host",
    "value" : "fv-az1773-824"
  }, {
    "name" : "thread",
    "value" : "2267-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests_api.test_bd"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 0,
        "skipped" : 36,
        "passed" : 2,
        "unknown" : 0,
        "total" : 38
      },
      "items" : [ {
        "uid" : "bb58bb12eebf9b34",
        "reportUrl" : "https://Areso.github.io/Oblache/158//#testresult/bb58bb12eebf9b34",
        "status" : "skipped",
        "statusDetails" : "XFAIL When using this method during a test run, the database may be in deleting status.\n\nAssertionError: Incorrect status code: 400",
        "time" : {
          "start" : 1726450055195,
          "stop" : 1726450055936,
          "duration" : 741
        }
      }, {
        "uid" : "9d9ded9fd6b69067",
        "reportUrl" : "https://Areso.github.io/Oblache/157//#testresult/9d9ded9fd6b69067",
        "status" : "skipped",
        "statusDetails" : "XFAIL When using this method during a test run, the database may be in deleting status.\n\nAssertionError: Incorrect status code: 400",
        "time" : {
          "start" : 1726364293530,
          "stop" : 1726364294535,
          "duration" : 1005
        }
      }, {
        "uid" : "dd5a9f8bf88749f3",
        "reportUrl" : "https://Areso.github.io/Oblache/156//#testresult/dd5a9f8bf88749f3",
        "status" : "skipped",
        "statusDetails" : "XFAIL When using this method during a test run, the database may be in deleting status.\n\nAssertionError: Incorrect status code: 400",
        "time" : {
          "start" : 1726276914061,
          "stop" : 1726276914789,
          "duration" : 728
        }
      }, {
        "uid" : "5a488acc6aae4447",
        "reportUrl" : "https://Areso.github.io/Oblache/155//#testresult/5a488acc6aae4447",
        "status" : "skipped",
        "statusDetails" : "XFAIL When using this method during a test run, the database may be in deleting status.\n\nAssertionError: Incorrect status code: 400",
        "time" : {
          "start" : 1726190585152,
          "stop" : 1726190585757,
          "duration" : 605
        }
      }, {
        "uid" : "f7197483e7b07d46",
        "reportUrl" : "https://Areso.github.io/Oblache/154//#testresult/f7197483e7b07d46",
        "status" : "skipped",
        "statusDetails" : "XFAIL When using this method during a test run, the database may be in deleting status.\n\nAssertionError: Incorrect status code: 400",
        "time" : {
          "start" : 1726104173676,
          "stop" : 1726104174697,
          "duration" : 1021
        }
      }, {
        "uid" : "229cf04b566ef4f2",
        "reportUrl" : "https://Areso.github.io/Oblache/153//#testresult/229cf04b566ef4f2",
        "status" : "skipped",
        "statusDetails" : "XFAIL When using this method during a test run, the database may be in deleting status.\n\nAssertionError: Incorrect status code: 400",
        "time" : {
          "start" : 1726017730121,
          "stop" : 1726017730754,
          "duration" : 633
        }
      }, {
        "uid" : "8b652a9d7bb3e608",
        "reportUrl" : "https://Areso.github.io/Oblache/152//#testresult/8b652a9d7bb3e608",
        "status" : "skipped",
        "statusDetails" : "XFAIL When using this method during a test run, the database may be in deleting status.\n\nAssertionError: Incorrect status code: 400",
        "time" : {
          "start" : 1725931387572,
          "stop" : 1725931388208,
          "duration" : 636
        }
      }, {
        "uid" : "9482473b6c22a07",
        "reportUrl" : "https://Areso.github.io/Oblache/12//#testresult/9482473b6c22a07",
        "status" : "skipped",
        "statusDetails" : "XFAIL When using this method during a test run, the database may be in deleting status.\n\nAssertionError: Incorrect status code: 400",
        "time" : {
          "start" : 1725892844626,
          "stop" : 1725892845258,
          "duration" : 632
        }
      }, {
        "uid" : "d85b77d46ccf781a",
        "reportUrl" : "https://Areso.github.io/Oblache/151//#testresult/d85b77d46ccf781a",
        "status" : "skipped",
        "statusDetails" : "XFAIL When using this method during a test run, the database may be in deleting status.\n\nAssertionError: Incorrect status code: 400",
        "time" : {
          "start" : 1725845231137,
          "stop" : 1725845231761,
          "duration" : 624
        }
      }, {
        "uid" : "568ffbc5b9ce6e74",
        "reportUrl" : "https://Areso.github.io/Oblache/11//#testresult/568ffbc5b9ce6e74",
        "status" : "skipped",
        "statusDetails" : "XFAIL When using this method during a test run, the database may be in deleting status.\n\nAssertionError: Incorrect status code: 400",
        "time" : {
          "start" : 1725789090068,
          "stop" : 1725789090697,
          "duration" : 629
        }
      }, {
        "uid" : "f3b971334424d50",
        "reportUrl" : "https://Areso.github.io/Oblache/150//#testresult/f3b971334424d50",
        "status" : "skipped",
        "statusDetails" : "XFAIL When using this method during a test run, the database may be in deleting status.\n\nAssertionError: Incorrect status code: 400",
        "time" : {
          "start" : 1725758891815,
          "stop" : 1725758892554,
          "duration" : 739
        }
      }, {
        "uid" : "2ea1caf0ed0c6ac0",
        "reportUrl" : "https://Areso.github.io/Oblache/149//#testresult/2ea1caf0ed0c6ac0",
        "status" : "skipped",
        "statusDetails" : "XFAIL When using this method during a test run, the database may be in deleting status.\n\nAssertionError: Incorrect status code: 400",
        "time" : {
          "start" : 1725672060238,
          "stop" : 1725672061272,
          "duration" : 1034
        }
      }, {
        "uid" : "aef32026167dd5dd",
        "reportUrl" : "https://Areso.github.io/Oblache/148//#testresult/aef32026167dd5dd",
        "status" : "skipped",
        "statusDetails" : "XFAIL When using this method during a test run, the database may be in deleting status.\n\nAssertionError: Incorrect status code: 400",
        "time" : {
          "start" : 1725585853303,
          "stop" : 1725585853932,
          "duration" : 629
        }
      }, {
        "uid" : "611a59c188753522",
        "reportUrl" : "https://Areso.github.io/Oblache/10//#testresult/611a59c188753522",
        "status" : "skipped",
        "statusDetails" : "XFAIL When using this method during a test run, the database may be in deleting status.\n\nAssertionError: Incorrect status code: 400",
        "time" : {
          "start" : 1725527901781,
          "stop" : 1725527902405,
          "duration" : 624
        }
      }, {
        "uid" : "7c1122be37534a4b",
        "reportUrl" : "https://Areso.github.io/Oblache/9//#testresult/7c1122be37534a4b",
        "status" : "passed",
        "statusDetails" : "XPASS When using this method during a test run, the database may be in deleting status.",
        "time" : {
          "start" : 1725526806325,
          "stop" : 1725526806963,
          "duration" : 638
        }
      }, {
        "uid" : "fa2aa6b4ceaed5e7",
        "reportUrl" : "https://Areso.github.io/Oblache/147//#testresult/fa2aa6b4ceaed5e7",
        "status" : "passed",
        "statusDetails" : "XPASS When using this method during a test run, the database may be in deleting status.",
        "time" : {
          "start" : 1725499527058,
          "stop" : 1725499527824,
          "duration" : 766
        }
      }, {
        "uid" : "b75b894f654ff4e2",
        "reportUrl" : "https://Areso.github.io/Oblache/146//#testresult/b75b894f654ff4e2",
        "status" : "skipped",
        "statusDetails" : "XFAIL When using this method during a test run, the database may be in deleting status.\n\nAssertionError: Incorrect status code",
        "time" : {
          "start" : 1725412944781,
          "stop" : 1725412945798,
          "duration" : 1017
        }
      }, {
        "uid" : "22394f83223ead04",
        "reportUrl" : "https://Areso.github.io/Oblache/145//#testresult/22394f83223ead04",
        "status" : "skipped",
        "statusDetails" : "XFAIL When using this method during a test run, the database may be in deleting status.\n\nAssertionError: Incorrect status code",
        "time" : {
          "start" : 1725326472792,
          "stop" : 1725326473805,
          "duration" : 1013
        }
      }, {
        "uid" : "4a3e77483edac659",
        "reportUrl" : "https://Areso.github.io/Oblache/144//#testresult/4a3e77483edac659",
        "status" : "skipped",
        "statusDetails" : "XFAIL When using this method during a test run, the database may be in deleting status.\n\nAssertionError: Incorrect status code",
        "time" : {
          "start" : 1725240200538,
          "stop" : 1725240201558,
          "duration" : 1020
        }
      }, {
        "uid" : "a87655c065812ec0",
        "reportUrl" : "https://Areso.github.io/Oblache/143//#testresult/a87655c065812ec0",
        "status" : "skipped",
        "statusDetails" : "XFAIL When using this method during a test run, the database may be in deleting status.\n\nAssertionError: Incorrect status code",
        "time" : {
          "start" : 1725154820048,
          "stop" : 1725154820770,
          "duration" : 722
        }
      } ]
    },
    "tags" : [ "@pytest.mark.xfail(reason='When using this method during a test run, the database may be in deleting status.')" ]
  },
  "source" : "abbaa4803039c0e8.json",
  "parameterValues" : [ ]
}